apps/scan/src/test/scala/org/lfdecentralizedtrust/splice/store/db/AcsSnapshotStoreTest.scala

```diff
@@ -449,12 +453,60 @@ class AcsSnapshotStoreTest
             )
           }
           _ <- store.insertNewSnapshot(None, DefaultMigrationId, timestamp1)
-          result <- queryRecursive(store, None, Vector.empty)
+          result <- queryRecursive(store, None, Vector.empty, Seq.empty, Seq.empty)
         } yield {
           result should be(contracts.map(_.contractId.contractId))
         }
       }
 
+      "paginate with filters by both party and template" in {
```

Comment:
Can you explain what exactly the test coverage is that you're adding here? Just a bit hard to see without looking through all existing tests